src/server/messages.ts

```diff
@@ -3,6 +3,7 @@ import { request } from 'http';
 import { createMultipart /*, createResponseMessage */} from './mime';
 import { parseString } from 'xml2js';
 import { ServerResult } from '../client/server_result';
+import { log } from '../logger';
```

Comment:
The changes you did here seem to be working just fine, but from a styling point of you you can alternatively do this:
```suggestion
import * as nisabaLogger from '../logger';
```
and call the logging functions below as
```typescript
nisabaLogger.info(...)
nisabaLogger.error(...)
nisabaLogger.debug(...)
```
instead of
```typescript
log('info', ...)
log('error', ...)
log('debug', ...)
```

Not sure whether we have preferences about this.  @ageorgou what do you think?

Reply:
I wouldn't particularly mind one way or another. My only issue with importing it as `nisabaLogger` is that it looks a bit like we're directly calling methods on an object, which is misleading. That's not necessarily a problem though (it could even be useful if we want to encourage ourselves to think about the interface in that way?) and a different name (e.g. `logging`) would also "fix" that.